dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/LoginController.java

```diff
@@ -223,16 +225,28 @@ public void loginByAuth2(@RequestParam String code, @RequestParam String provide
             requestParamsMap.put("grant_type", "authorization_code");
             requestParamsMap.put("redirect_uri",
                     String.format("%s?provider=%s", oAuth2ClientProperties.getRedirectUri(), provider));
-            String tokenJsonStr = OkHttpUtils.post(oAuth2ClientProperties.getTokenUri(), tokenRequestHeader,
-                    requestParamsMap, requestBody);
+            OkHttpRequestHeaders okHttpRequestHeadersPost = new OkHttpRequestHeaders();
+            okHttpRequestHeadersPost.setHeaders(tokenRequestHeader);
+            okHttpRequestHeadersPost.setOkHttpRequestHeaderContentType(OkHttpRequestHeaderContentType.APPLICATION_JSON);
+            int timeout = 30000;
```

Comment:
It's better to use constant but not magic number.

Reply:
Fixed.